node/src/main/resources/reference.conf

```diff
@@ -45,8 +45,11 @@ rnode {
     # Maximum number of peers allowed to connect to the node
     max-connections = 20
 
-    # Maximum size of message that can be sent via transport layer
-    max-message-size = 256K
+    # Maximum size of message that can be received via transport layer
+    max-message-size = 2M
```

Comment:
wowowow why 2M?

Reply:
@rabbitonweb Why 256K? :)
This is the max incoming message size a node will accept. Why would a node accept only messages that are smaller than 256K? The default gRPC max message size is 4MB. So 2MB is 50% of that and in my opinion a quite fair message size.